### PR TITLE
add total number of words learned in Profile status

### PR DESCRIPTION
**Pull request by kian1213**
Hi sir, please review my pull request, Thanks po

### CODE REVIEW

**Comment by codegracie** [comment]
Ok

**Comment by kian1213** [comment]
Used scope for the activity query.

File: app/models/user.rb [hunk 1]
```diff
@@ -78,6 +77,17 @@ def following? other_user
     following.include? other_user
   end
 
+  def total_words_learned
```
Comment (dieunb) [inline app/models/user.rb]: you can refactor to short and better?

Comment (kian1213) [inline app/models/user.rb]: Ok sir, will do po

**Pull request merged**